### PR TITLE
perf: improve parsing form-data

### PR DESCRIPTION
**Pull request by tsctx**
Well, it can be further optimized, but that would only increase the code, not much of a benefit.

### CODE REVIEW

**Comment by tsctx** [comment]
The failure of ci has nothing to do with my changes.

File: lib/core/util.js [hunk 1]
```diff
@@ -449,6 +450,13 @@ function toUSVString (val) {
   return hasToWellFormed ? `${val}`.toWellFormed() : nodeUtil.toUSVString(val)
 }
 
+/**
+ * @param {string} val
+ */
+function isUSVString (val) {
```
Comment (KhafraDev) [inline lib/core/util.js:456]: please add a TODO to move this to webidl

**Review by KhafraDev — APPROVED** [review]

File: lib/core/util.js (hunk 1 above)
Comment (tsctx) [reply lib/core/util.js:456]:
```suggestion
// TODO: move this to webidl 
function isUSVString (val) {
```

File: lib/core/util.js [hunk 2]
```diff
@@ -449,6 +450,14 @@ function toUSVString (val) {
   return hasToWellFormed ? `${val}`.toWellFormed() : nodeUtil.toUSVString(val)
 }
 
+/**
+ * @param {string} val
+ */
+// TODO: move this to webidl
```
Comment (tsctx) [inline lib/core/util.js:456]:
```suggestion
// TODO: move this to webidl
```

File: lib/web/fetch/formdata-parser.js [hunk 3]
```diff
@@ -15,6 +15,18 @@ const filenameBuffer = Buffer.from('; filename')
 const dd = Buffer.from('--')
 const ddcrlf = Buffer.from('--\r\n')
 
+/**
+ * @param {string} chars
+ */
+function isAsciiString (chars) {
+  for (let i = 0; i < chars.length; ++i) {
+    if ((chars.charCodeAt(i) & ~0x7F) !== 0) {
```
Comment (Uzlopak) [inline lib/web/fetch/formdata-parser.js:23]: Wonder if -0x80 is the same

Comment (tsctx) [reply lib/web/fetch/formdata-parser.js:23]: Same, but I think this one is more readable.

**Review by KhafraDev — APPROVED** [review]

**Pull request merged**